schema/transportation/segment.yaml

```diff
@@ -51,6 +57,71 @@ properties:
         default: []
 "$defs":
   propertyDefinitions:
+    destinationLabelType:
+      description: >-
+        The type of object of the destination label.
+      type: string
+      enum: 
+      - place
+      - street
+      - country 
+      - route
+    destinations:
+      description: >-
+        Information about what destinations can be reached if following this segment. 
+        Includes references to the segments from which it is needed to transition 
+        onto this segment to reach the destinations.
```

Comment:
Something that's a bit "off" to me is that, depending on what part of the schema and documentation you read, the schema doesn't seem to be sure about whether `destinations` models *destinations* themselves, or whether it models the *signs* that point to destinations.

On reflection I think it's the signs, since the purpose of `destinations` is to model the signs a driver or traveller actually sees.

Can we have a go at making the documentation consistent?

I would be tempted to start with a description along the following lines:

```
Describes objects that can be reached by following a transportation segment in the same way those
objects are described on signposts or ground writing that a traveller following the segment in would
observe in the real world. This allows navigation systems to refer to signs and observable writing
that a traveller actually sees.
```